### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vladgusakovskiy**

### CODE REVIEW

**Review by max-kravchenko — APPROVED** [review]
Great!

File: src/makeCalculator.js [hunk 1]
```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, value) {
+      operation.call(this, value);
```
Comment (AlinaMatuschak) [inline src/makeCalculator.js:44]: 🔥

File: src/makeCalculator.js [hunk 2]
```diff
@@ -37,7 +37,40 @@
...
+
+    divide(value) {
+      this.result /= value;
+    },
+  };
+
+  return calculator;
 }
 
+makeCalculator();
```
Comment (AlinaMatuschak) [inline src/makeCalculator.js:75]: Unnecessary

**Review by AlinaMatuschak — APPROVED** [review]
Good work 👍

**Pull request closed without merging**